pom.xml

```diff
@@ -224,6 +224,12 @@ under the License.
 				<version>3.2.1</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-math3</artifactId>
+				<version>3.5</version>
+			</dependency>
```

Comment:
I didn't found flink-dist/NOTICE and flink-dist/LICENSE files, there are NOTICE and LICENSE in flink root directory, but i didn't found the right place to add something related to commons-math3, so i would just leave this to you, Till.